src/routes/histogram.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from "express";
+import { projection } from "../db/schema/projection.js";
+
+const route = Router();
+
+route.get("/:column/histogram", async (req, res) => {
+  const columnName = req.params.column;
+
+  console.log({ columnName });
+
+  await req.db.select().from(projection);
```

Comment:
The format of this endpoint is about to get fun with CSV Headers coming in and me trying to map that back to data I've already converted to database columns.